test/lib/absinthe/type/custom/type_b

```diff
@@ -0,0 +1,8 @@
+defmodule Absinthe.Type.Custom.TypeB
+  use Hyr.Command
```

Comment:
What is this?

Reply:
my bad, copy and paste error

Reply:
and I don't use it as an input_object, but the import is what matters

Reply:
should be all set now